core/templates/domain/utilities/language-util.service.ts

```diff
@@ -145,6 +148,28 @@ export class LanguageUtilService {
     const language = this.getSupportedAudioLanguages()[audioLanguageCode];
     return language ? language.description : null;
   }
+
+  getContentLanguageDirection(languageCode: string): string {
+    let languageDirection = 'auto'; // Default auto for invalid languageCode
+    this.SUPPORTED_CONTENT_LANGUAGES.forEach((lang: ContentLanguage) => {
+      if (lang.code === languageCode) {
+        languageDirection = lang.direction;
+      }
+    });
+    return languageDirection;
+  }
+
+  getAudioLanguageDirection(languageCode: string): string {
```

Comment:
hmm.. a direction for audio language sounds confusing. (Considering bother the language list will have the same direction for the same language list.) I would suggest having one method which will loop through both the list and returns the direction wherever found. `getLanguageDirection`. Let me know what you think?

Reply:
The reasoning for this is here: https://github.com/oppia/oppia/pull/11499#discussion_r553500920
The only downside to combining these again is that `translation-language.service.ts` will then use both audio language and content language.

Reply:
I decided to combine these. Let me know if you agree.